datamodels/core/src/main/java/net/fhirfactory/pegacorn/core/file/common/DateUtils.java

```diff
@@ -0,0 +1,184 @@
+package net.fhirfactory.pegacorn.core.file.common;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Date;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utility class for converting and formatting dates.
+ * 
+ * @author Brendan Douglas
+ *
+ */
+public class DateUtils {
+    
+    private static final Logger logger = LoggerFactory.getLogger(DateUtils.class);
+    
+    public static final String DD_MMM_YY = "dd-MMM-yy";
+    public static final String DD_MMM_YYYY = "dd-MMM-yyyy";
+    public static final String DD_MMM_YY_HH_MM_SS = "dd-MMM-yy HH:mm:ss";
+    public static final String D_M_YYYY_H_MM_SS = "d/M/yyyy h:mm:ss";
+    public static final String YYYY_MM_DD_T_HH_MM_SS_Z_INPUT = "yyyy-MM-dd'T'HH:mm:ssz";
+    public static final String YYYY_MM_DD_T_HH_MM_SS_INPUT = "yyyy-MM-dd'T'HH:mm:ss";
+    public static final String YYYY_MM_DD_T_HH_MM_SS_Z_OUTPUT = "yyyy-MM-dd'T'HH:mm:ssZ";
+    public static final String YYYY_MM_DD_T_HH_MM_SS_ZOFFSET = "yyyy-MM-dd'T'HH:mm:ss[+-]hh:mm";    
+    public static final String YYYY_MM_DD = "yyyy-MM-dd";
+    public static final String DD_MM_YYYY = "dd/MM/yyyy";
```

Comment:
These set formats work well for us but we probably shouldn't use these if we want it to be suitable for an international audience.